package.json

```diff
@@ -24,7 +24,7 @@
     "build:server": "babel --config babel.config.js -x .js,.jsx -d dist/server src/client src/server/page.html.jsx",
```

Comment:
this fails if /dist/server does not exist, maybe a `prebuild` before build:server?

Reply:
👍 fixed with a `prebuild:server` command.